src/rules/noUnusedVariableRule.ts

```diff
@@ -220,7 +228,7 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
             const variableName = (<ts.Identifier> node.name).text;
 
             if (Lint.hasModifier(modifiers, ts.SyntaxKind.PrivateKeyword)) {
-                this.validateReferencesForVariable(variableName, node.name.getStart());
+                this.validateReferencesForVariable("method", variableName, node.name.getStart());
```

Comment:
Oops!  I missed this one!  These all started out as strings before they got moved to static strings on the class